planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/schema/static_obstacle_avoidance.schema.json

```diff
@@ -22,14 +22,10 @@
           "description": "Path generation method.",
           "default": "shift_line_base"
         },
-        "use_adjacent_lane": {
-          "type": "boolean",
-          "description": "Extend avoidance trajectory to adjacent lanes that has same direction. If false, avoidance only happen in current lane.",
-          "default": "true"
-        },
-        "use_opposite_lane": {
-          "type": "boolean",
-          "description": "Extend avoidance trajectory to opposite direction lane. `use_adjacent_lane` must be `true` to take effects.",
+        "use_lane_type": {
+          "type": "string",
+          "enum": ["current_lane", "same_direction_lane", "opposite_direction_lane"],
+          "description": "Drivable lane configuration.",
           "default": "true"
```

Comment:
```suggestion
          "default": "current_lane"
```

Reply:
Ahhhhhhhh.....

Reply:
I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/7715/commits/65ace5a8ca733bf6a020abf85fcf90c098520b7f.